MAINTAINERS.md

```diff
@@ -0,0 +1,13 @@
+# Project Maintainers
```

Comment:
Could you add @leseb and myself?

Reply:
Thoughts on this? @ashwinb @raghotham

Reply:
I've removed that file to focus on CODEOWNERS only